### ISSUE TITLE
[Chromaticraft V33a] Puzzle Barrier can be phased through

### ISSUE THREAD

**Issue by Megadeadly** [ReikaKalseki/Reika_Mods_Issues#3370]
I'm not familiar yet with what structure this is, or at the moment how to properly complete it, but the barrier in the floor can be slowly phased through by sneaking so you aren't lifted out, and waiting. You will gradually drop a little bit from time to time until finally bypassing the barrier entirely. I don't know if this is actually useful in bypassing the puzzle, as I didn't want to break the structure entirely and skip the puzzle, but doubt this is desired regardless. On a small side note, I wouldn't mind being informed on whether or not the red barrier surrounding the structure is supposed to indicate that I can't complete it currently.

This was done on an updated and modified version of the SetGameDifficulty Reika modpack (Chromaticraft isn't modified in any way that I know of). I have not tried replicating this on a fresh installation with no extra mods.

TL;DR Impassable barrier can be slowly passed through by sneaking and waiting long enough.

Pictures to identify the structure in question and showing I'm inside.

<details><summary>Maybe Spoiler?</summary>
<p>

![Inside](https://github.com/ReikaKalseki/Reika_Mods_Issues/assets/61374891/f0b33355-5f0b-4e77-ad35-02d14798f65b)

![Outside](https://github.com/ReikaKalseki/Reika_Mods_Issues/assets/61374891/b3a97aab-a6c0-4767-a1f4-98f9e76e6802)
</p>
</details>

**Comment by Megadeadly** [comment]
Oh uh, it also appears that if a Thaumcraft node is too close to the structure, if you leave and come back, when the structure reloads, the node gets boosted/duplicated into itself. A couple trips back and forth and the node on top of the structure is well past the thousands of each aspect it originally contained. Not really an issue, more abusably hilarious than anything, but gotta mention it. Also, my eyes burn from looking at it.

**Comment by Gronglegrowth** [comment]
That node issue is actually because of it being in a Rainbow Forest, see #2120 for a lot more details (I came across this while looking for that, actually).

Also IIRC, that structure cannot be completed at all before you're progressed far enough.

**Comment by Megadeadly** [comment]
Alright, thanks for the info.
